wagtail/admin/templates/wagtailadmin/shared/locale_selector.html

```diff
@@ -1,27 +1,20 @@
-{# Based on listing/_button_with_dropdown.html #}
 {% load wagtailadmin_tags %}
+
+{% fragment as locale_label %}
+    {% icon name="site" classname="middle" %}
+    {{ locale.get_display_name }}
+{% endfragment %}
```

Comment:
This is reused both for the non-interactive version of the locale selector, and as label for the dropdown when in use.